cores/esp8266/cont_util.c

```diff
@@ -38,13 +38,13 @@ void ICACHE_RAM_ATTR cont_init(cont_t* cont) {
     }
 }
 
-int ICACHE_RAM_ATTR cont_check(cont_t* cont) {
+int cont_check(cont_t* cont) {
     if(cont->stack_guard1 != CONT_STACKGUARD || cont->stack_guard2 != CONT_STACKGUARD) return 1;
 
     return 0;
 }
 
-int ICACHE_RAM_ATTR cont_get_free_stack(cont_t* cont) {
+int cont_get_free_stack(cont_t* cont) {
```

Comment:
I intend to call this from an ISR for statistical purposes (on my todo list for my application), so I'd prefer to keep it in RAM.